README.md

```diff
@@ -14,48 +14,62 @@ service, and helpers, provide a way to format dates, numbers, strings messages,
 **<http://formatjs.io/ember/>**
 
 ## Requirements
-* Ember-cli >= 0.1.5
+* Ember-cli >= 0.2.0
 * Ember >= 1.10.x
 * HTMLBars
 
+## Ember-Intl 2.0
+
+This README is targetting and explaining the 2.0 API which differs from previous versions.  If you are looking for the 1.3.0 documentation, please check [here](https://github.com/yahoo/ember-intl/blob/1.3.0/README.md).
+
+
 ## Installation
-* `ember install ember-intl` (or `ember install:addon ember-intl` for ember-cli < v0.2.3) 
+* `ember install ember-intl` (or `ember install:addon ember-intl` for ember-cli < v0.2.3)
 * If you are targeting a browser that doesn't support the native Intl API (such as Safari or PhantomJS), you need to load the shim.  The [Intl.JS polyfill](https://github.com/andyearnshaw/Intl.js/) is automatically added into your asset distribution folder, so you need to add the following to your index.html:
 
 ```html
 <script src="/assets/intl/polyfill/Intl.complete.js"></script>
```

Comment:
Add comments here around Intl.complete being a bad idea.  Instead, should bring in Intl.min along with the locales you app targets